src/index.html

```diff
@@ -11,6 +11,51 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form 
+      action="#"
+      method="get"
+      class="search-form"
+      data-qa="big"
+    >
+      <label for="bigSearchField">
+        <img src="images/Search.svg"
+          alt="imageSearch"
+          class="icon"
+        >
+        <input type="search"
+        name="bigSearchField"
+        id="bigSearchField"
+        class="big-search search"
+        data-qa="keypress"
+        placeholder="Try &quot;Los Angeles&quot;"
+        >
```

Comment:
```suggestion
        <img src="images/Search.svg"
          alt="imageSearch"
          class="icon"
        >
        
        <input type="search"
          name="bigSearchField"
          id="bigSearchField"
          class="big-search search"
          data-qa="keypress"
          placeholder="Try &quot;Los Angeles&quot;"
        >
```
fix the code style all over the page